cbw_api_toolbox/cbw_api.py

```diff
@@ -3,19 +3,18 @@
 import json
 import logging
 import sys
-
+import datetime
 import requests
 from requests.exceptions import ProxyError, SSLError, RetryError, InvalidHeader, MissingSchema
 from urllib3.exceptions import NewConnectionError, MaxRetryError
-
+import xlsxwriter
 from cbw_api_toolbox import API_DEFAULT_URL
 from cbw_api_toolbox.__routes__ import ROUTE_SERVERS, ROUTE_PING, ROUTE_REMOTE_ACCESSES
 from cbw_api_toolbox.cbw_auth import CBWAuth
 from cbw_api_toolbox.cbw_objects.cbw_server import CBWServer
 from cbw_api_toolbox.cbw_objects.cbw_remote_access import CBWRemoteAccess
 from cbw_api_toolbox.cbw_parser import CBWParser
 
-
```

Comment:
why?

Reply:
No problem in travis

Reply:
RTFM : https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
RTFM french please

Reply:
http://nguyen.univ-tln.fr/share/Python/pep8.pdf